packages/react/src/components/DatePicker/plugins/fixEventsPlugin.js

```diff
@@ -109,9 +109,10 @@ export default (config) => (fp) => {
         if (inputTo.value) {
           fp.setDate(
             [inputFrom.value, inputTo.value],
-            true,
+            false,
```

Comment:
Does previously sending the second value as true , was forcefully firing onChange event ?